src/open_samus_returns_rando/pickups/pickup.py

```diff
@@ -226,7 +217,10 @@ def patch_minimap(self, editor: PatcherEditor, scenario_name: str, actor_name: s
                 else:
                     pickup_tile_icon.icon = "itemenabled"
             else:
-                pickup_tile_icon.icon = self.pickup["map_icon"]
+                first_model_name = self.pickup["model"][0]
+                icon = self.pickup.get("map_icon", None)
+                icon = get_map_icon_data(first_model_name) if icon is None else icon
```

Comment:
I assume this reassignment is fine. It just looks weird but I understand what's going on.

Reply:
I saved a total of 1 line with that!
...I changed it :p